Runtime/Scripts/GltFast.cs

```diff
@@ -575,7 +575,7 @@ enum ImageFormat {
                             images[dl.Key] = txt;
                         }
                     } else {
-                        Debug.LogError(www.error);
```

Comment:
Please leave this as `LogError` for now, as test have to fail whenever this happens. 

A better long-term solution is a custom loading strategy (where you can define the behavior) and a proper loading report (without logging required).